src/BloomTests/web/EnhancedImageServerTests.cs

```diff
@@ -173,6 +173,64 @@ public void CanRetrieveContentOfFakeTempFile_ButOnlyUntilDisposed()
 			}
 		}
 
+		[Test]
+		public void CanRetrieveContentOfFakeTempFile_WhenFolderContainsAmpersand_ViaJavaScript()
+		{
+			using (var server = CreateImageServer())
+			{
+				var ampSubfolder = Path.Combine(_folder.Path, "Mom & Dad");
```

Comment:
For a real torture test you could try a title like "Using &lt;, &gt;, & &amp; in HTML"